### ISSUE TITLE
[bug] GriefClientPro - When hosting a game and the default for [Visible] is unchecked it causes syncing issues

### ISSUE THREAD

**Issue by L-a-n-g-o-l-i-e-r-s** [Hellsing/ModAPI-Mods#2]
Hi there,

When I go to host a game, when arriving having this option toggled off by default all of the suitcases are not outside the plane and trees/shrubs vanish/disolve as I get closer to them.

This is only partially fixed if someone joins after toggling visible on, shrubs will still pop in and out and sometimes the trees, bodies of water like ponds never re-appear, yet you still float in them. (Ocean isn't effected nor lakes.) One example is the tiny boat pond/water fall and there is an underwater cave in it, it has some weird platforms with roaps high up in the pond (2).

Also importantly that it seems to others ability to see your text in chat when you host with visible unticked by default, when ticked after the ability for others to see that text chat never returns, they can however hear your microphone if you use the walky talky. (When hosting with it off by default and ticked ON for visibility.)

Basically, you have to join a SP or MP game, toggle that option off before hosting your own to avoid that, you can probably toggle it off after to avoid the issue.

(This issue only exists if you host MP game, not SP.)

Thanks

**Comment by L-a-n-g-o-l-i-e-r-s** [comment]
I changed "public static bool Visible = false;" to true in menu.cs from the source files, compiled it and it resolved the issue. (This is a matter of taste really, if you host games you should do this and after starting the game you can toggle it off and not suffer the issues listed above.)

Closing this issue, thanks.
